src/components/RewardsModal/RewardsModal.tsx

```diff
@@ -0,0 +1,81 @@
+import { Button, ButtonColor } from '../Button/Button'
+import { shortenAddress } from '../../utils/shortenAddress'
+import { Modal } from '../Modal/Modal'
+import { format } from '../../utils'
+import { useEffect, useState } from 'react'
+import { getConnection } from '../../utils/useConnect'
+import { femtoKiltToDigits } from '../../utils/conversion'
+
+export interface Props {
+  accountAddress: string
+  rewards: number
+  onConfirm: () => void
+  closeModal: () => void
+}
+
+export const RewardModal: React.FC<Props> = ({
+  accountAddress,
+  rewards,
+  onConfirm,
+  closeModal,
+}) => {
+  const shortAddress = shortenAddress(accountAddress)
+  const [fee, setFee] = useState<number>(0.0003)
+
+  useEffect(() => {
+    const getFee = async () => {
+      const api = await getConnection()
+
+      const feeInFemto = (
+        await api.tx.utility
+          .batch([
+            api.tx.parachainStaking.incrementDelegatorRewards(),
+            api.tx.parachainStaking.claimRewards(),
+          ])
+          .paymentInfo(accountAddress)
```

Comment:
ok one last suggestion: 
```suggestion
        await claimRewards().then(tx =>
          tx.paymentInfo(accountAddress))
```

Reply:
b23d52aaaa27584a1a8f564b392fb1d34a6f3da4

Reply:
oops: 35a564a756aacac5422352e91687873fff05ebb6